src/omlt/io/__init__.py

```diff
@@ -1,5 +1,8 @@
-from omlt.io.onnx import (
-    load_onnx_neural_network,
-    load_onnx_neural_network_with_bounds,
-    write_onnx_model_with_bounds,
-)
+from omlt.dependencies import onnx_available
```

Comment:
what are your thoughts on exposing the keras reader here if it is available? As is: 
`from omlt.io import load_keras_sequential` 
will not work. We need to update the README.rst page which uses this import otherwise.